src/test/resources/loose-war-with-loose-jar/ejb-war/build.gradle

```diff
@@ -25,7 +25,7 @@ dependencies {
 }
 
 war {
-    archiveName = baseName + '.' + extension
+    archiveFileName = getArchiveBaseName.get() + '.' + getArchiveExtension().get()
```

Comment:
I think this is missing the `()` after `getArchiveBaseName`.

Reply:
Thanks, I missed that. Updated the PR.